biopet-utils/src/main/scala/nl/lumc/sasc/biopet/utils/summary/db/Schema.scala

```diff
@@ -0,0 +1,146 @@
+package nl.lumc.sasc.biopet.utils.summary.db
+
+import java.sql.Date
+
+import slick.driver.H2Driver.api._
+
+/**
+ * Created by pjvan_thof on 27-1-17.
+ */
+object Schema {
+
+  case class Run(id: Int, name: String, outputDir: String, version: String, commitHash: String,
```

Comment:
Empty string I never tested but I assume this just works. If the database implementation does not accept this it will raise an error but in theory a empty string is still a string.

The schema here is very basic because a lot of those things are not working in sqlite. If using this for in an other implementation we can extends the schema's to a more specific implementation.

For a version there is no regex because not all tools give the same kind of version numbers.